Web/src/components/games/PandaSequence/GameDisplay.tsx

```diff
@@ -35,6 +37,23 @@ const GameContainer = styled.div`
   text-shadow: 3px 3px 0px rgba(0, 0, 0, 0.1);
 `;
 
+const WobbleAnimation = keyframes`${wobble}`;
+
+const DisplayBalloon = styled(BalloonSVG)`
+  width: 70px;
+  margin: 12px;
+  animation: ${(tags: {duration: number}) => (tags.duration === 0
```

Comment:
I think `props` is a better name than `tags`

Reply:
Ah yeap I used tags because props was used in the same line previously, but now it's gone lemme change back :D